Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerVisualizationProfile.asset

```diff
@@ -51,3 +51,23 @@ MonoBehaviour:
     controllerVisualizationType:
       reference: Microsoft.MixedReality.Toolkit.Input.WindowsMixedRealityControllerVisualizer,
         Microsoft.MixedReality.Toolkit.SDK
+  - description: Oculus XRSDK Touch Controller
```

Comment:
I think these can be removed, since there doesn't seem to be any overridden data here. Can just use the defaults!